### ISSUE TITLE
--CPUs does not limit vsearch number of threads / CPUs

### ISSUE THREAD

**Issue by hmontenegro** [HRGV/phyloFlash#18]
While running

    phyloFlash_makedb.pl --remote

I noticed barrnap-HGV was running on all 32 cores available. I interrupted the script and ran it again with:

    phyloFlash_makedb.pl --remote --CPUs 4

This time barrnap-HGV ran with 4 threads, but then vsearch ran with 32 cores, so --CPUs is not being passed to vsearch.

**Comment by HRGV** [comment]
should be fixed, thanks for the helpful bug report!
Cheers
Harald and Brandon

**Comment by hmontenegro** [comment]
Thanks for the quick fix. After I wrote yesterday I noticed BBTools were using too many threads as well, thanks for fixing this also - but shouldn't $cpus be passed to the subs involving bbmap/bbduk/etc? E.g. sub mask_repeats on lines 174 and 243; sub univec_trim on lines 177 and 243; sub bbmap_db on lines 198 and 281 of phyloFlash_makedb.pl?
Also, mafft has a --threads option, but in this case the default is 1, so there is no overuse.
Again, thanks, and cheers
